src/batched/KokkosBatched_Gemv_Serial_Internal.hpp

```diff
@@ -61,7 +61,9 @@ namespace KokkosBatched {
           ValueType t(0);
           const ValueType *__restrict__ tA = (A + i*as0);
 
-              
+#if defined(KOKKOS_ENABLE_PRAGMA_IVDEP)
```

Comment:
@kyungjoo-kim Can #pragma unroll be nested within #pragma ivdep? I'm asking just in case both these options happen to get enabled at the same time, if this is not allowed will need more #if guard magic to select the preference.